control-plane/subcommand/sync-catalog/command.go

```diff
@@ -138,6 +143,10 @@ func (c *Command) init() {
 			"\"debug\", \"info\", \"warn\", and \"error\".")
 	c.flags.BoolVar(&c.flagLogJSON, "log-json", false,
 		"Enable or disable JSON output format for logging.")
+	c.flags.StringVar(&c.flagPurgeK8SServicesFromNode, "purge-k8s-services-from-node", "",
+		"Purge all K8S services registered in Consul under the node name.")
+	c.flags.StringVar(&c.flagFilter, "filter", "",
+		"Specifies the expression used to filter the queries results for the node.")
```

Comment:
Since we use the [List Services for Node](https://developer.hashicorp.com/consul/api-docs/catalog#list-services-for-node)API, the filter syntax is the same to that [API](https://developer.hashicorp.com/consul/api-docs/catalog#filtering-4), should I highlight this somewhere in the flag description?

Reply:
I added the details in the flag explanation